.gitignore

```diff
@@ -22,3 +22,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+newrelic_agent.log
+/server/newrelic_agent.log
```

Comment:
Can you please clarify what you are trying to pull here?

Reply:
no